src/Compilers/CSharp/Test/Semantic/Semantics/IteratorTests.cs

```diff
@@ -61,6 +62,67 @@ IEnumerable<int> I()
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void BasicIterators_Async()
+        {
+            var source = """
+                using System.Collections.Generic;
+                using System.Threading.Tasks;
+                
+                class Test
+                {
+                    async IAsyncEnumerable<int> I()
```

Comment:
>async IAsyncEnumerable<int> I() [](http://example.com/codeflow?start=20&length=31)

Should a regular iterator be tested as the next test is doing?

Reply:
Already tested in a test above (added public code path check there)